Composer/packages/client/src/pages/language-generation/table-view.tsx

```diff
@@ -17,42 +17,34 @@ import formatMessage from 'format-message';
 import { NeutralColors, FontSizes } from '@uifabric/fluent-theme';
 import { RouteComponentProps } from '@reach/router';
 import { LgTemplate } from '@bfc/shared';
-import { Async } from 'office-ui-fabric-react/lib/Utilities';
 
 import { StoreContext } from '../../store';
 import { increaseNameUtilNotExist } from '../../utils/lgUtil';
 import { navigateTo } from '../../utils';
 import { actionButton, formCell } from '../language-understanding/styles';
+import { useShell } from '../../shell';
 
 interface TableViewProps extends RouteComponentProps<{}> {
   dialogId: string;
 }
 
 const TableView: React.FC<TableViewProps> = props => {
   const { state, actions } = useContext(StoreContext);
+  const { api } = useShell();
+  const { announce } = api;
```

Comment:
This doesn't make sense to use here. This component should invoke the actions directly off of context.